src/content/blog/val-town-newsletter-15.md

```diff
@@ -0,0 +1,219 @@
+- [@saolsen](https://www.val.town/u/saolsen) created a Deno script that will [sync your vals to GitHub](https://www.val.town/v/saolsen/git_sync)
+- [@pomdtr](https://www.val.town/u/pomdtr) ported [blakeembrey/sql-template-tag](https://github.com/blakeembrey/sql-template-tag) to Val Town to simplify [building safe SQL queries](https://www.val.town/v/pomdtr/sql)
+- [@saolsen](https://www.val.town/u/saolsen) experimented with [writing tests with Jest ](https://www.val.town/v/saolsen/tiny_jest_example)
+- [@pomdtr](https://www.val.town/u/pomdtr) found a way to [serve static content by using val's readme]()https://www.val.town/v/pomdtr/notebook
+- [@stevekrouse](https://www.val.town/u/stevekrouse) wrote a whole fullstack (sqlite, backend js, frontend js, html) app in a [single val](https://www.val.town/v/stevekrouse/backend_in_a_file)
+- [@saolsen](https://www.val.town/u/saolsen) wrote a useful [val that can automatically prune multiple versions of other vals](https://www.val.town/v/saolsen/prune_val)
+- [@nbbaier](https://www.val.town/u/nbbaier) created a [val that can generate SQL](https://www.val.town/v/nbbaier/sqliteWriter) using [prisma-gpt](https://github.com/aliyeysides/prisma-gpt)
+- [@nbbaier](https://www.val.town/u/nbbaier) made a val that [generates readme for vals automatically](https://www.val.town/v/nbbaier/readmeGPT) using GPT
+- [@nbbaier](https://www.val.town/u/nbbaier) wrote a [Perplexity AI API wrapper val](https://www.val.town/v/nbbaier/perplextiyAPI)
+- [@pomdtr](https://www.val.town/u/pomdtr) continued his work on [lowdb adapter for Val Town](https://www.val.town/v/pomdtr/lowdb) allowing to use blobs as type-safe databases
+- [@joey](https://www.val.town/u/joey) made it easier to [use blobs as arrays](https://www.val.town/v/joey/BlobArray)
+- [@deepfates](https://www.val.town/u/deepfates) wrote a great [article on building a face swapping app with Val Town, HTMX, and Replicate]
+- [@tom_og](https://www.val.town/u/tomcritchlow) made a val that [generates open graph images with code](https://www.val.town/v/tomcritchlow/tom_og)
+- [@tmcw](https://www.val.town/u/tmcw) put together a [val that collects stats about his Figma plugins](https://www.val.town/v/tmcw/placemarkGlobeMonitor) that can be displayed as a [fancy graph](https://observablehq.com/@tmcw/placemark-figma-plugin-stats)
+- [@glommer](https://www.val.town/u/glommer) wrote an [amazing article on Turso blog](https://blog.turso.tech/i-needed-a-simple-discord-bot.-val-town-made-it-fun!-bf8339ab) documenting his adventure making a Discord bot using Val Town
+- [@stevekrouse](https://www.val.town/u/stevekrouse) wrote a whole fullstack (sqlite, backend js, frontend js, html) app in a [single val](https://www.val.town/v/stevekrouse/backend_in_a_file)
+- [@nbbaier](https://www.val.town/u/nbbaier) made a useful [utility val](https://www.val.town/v/nbbaier/sqliteDump) that generates SQL statements to dump the data and schema of tables in a SQLite database 
+
+---
+
+To organize:
```

Comment:
@stevekrouse I added all relevant cool vals from the "to organize" section, including the ones from Twitter. Not sure how to handle the rest of Twitter stuff.

Reply:
I don't understand your confusion. For example:

https://x.com/deepfates/status/1751015063912399126?s=20 - talk about how the person made an awesome guide to show you how to make a faceswapping app using ai and val town

Reply:
The ones that I'm not sure how to include:
- https://twitter.com/stevedylandev/status/1745667977393910096 - There's nothing to link, no actual val, just a random screenshot of some app that is calling a Val Town example.
- https://twitter.com/nbbaier/status/1744847782484070400 - Not sure what is going on here either.

The rest has been included in cool vals.

Reply:
https://twitter.com/stevedylandev/status/1745667977393910096  -> link to the tweet, besides that it should be similar to the rest

https://twitter.com/nbbaier/status/1744847782484070400  -> we can skip it